charts/cbcontainers-agent/cbcontainers-agent-chart/templates/containers-agent.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: operator.containers.carbonblack.io/v1
 kind: CBContainersAgent
 metadata:
   name: cbcontainers-agent
+  namespace: {{ default "cbcontainers-dataplane" .Values.agentNamespace }}
```

Comment:
@asankov-cb 
The CBContainersAgent is a cluster scoped resource, it doesn't have a namespace,
so putting this under metadata is confusing.
I think it can stay under spec.
Also, maybe it's not important cause no used the field yet, but it's a breaking change

Reply:
yes, I agree. I think that's why I initially put it under `spec` but then I moved it under `metadata`. Will close this PR.